patches/telemetry.diff

```diff
@@ -103,14 +138,6 @@ Index: code-server/lib/vscode/src/vs/workbench/services/telemetry/browser/teleme
 -				commonProperties: resolveWorkbenchCommonProperties(storageService, productService.commit, productService.version, isInternal, environmentService.remoteAuthority, productService.embedderIdentifier, productService.removeTelemetryMachineId, environmentService.options && environmentService.options.resolveCommonTelemetryProperties),
 -				sendErrorTelemetry: this.sendErrorTelemetry,
 -			};
--			this.impl = this._register(new BaseTelemetryService(config, configurationService, productService));
--
--			if (getTelemetryLevel(configurationService) !== TelemetryLevel.NONE) {
--				// If we cannot fetch the endpoint it means it is down and we should not send any telemetry.
--				// This is most likely due to ad blockers
--				fetch(telemetryEndpointUrl, { method: 'POST' }).catch(err => {
--					this.impl = NullTelemetryService;
--				});
```

Comment:
I actually have no idea what happened with this segment in the original version. I have not found any commit where this was removed or changed. Maybe sbd. finds the reason...